server/src/main/java/org/hyperskill/community/flashcards/common/exception/ResourceAlreadyExists.java

```diff
@@ -0,0 +1,12 @@
+package org.hyperskill.community.flashcards.common.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT)
+public class ResourceAlreadyExists extends RuntimeException {
+
```

Comment:
I have never used or seen used 409. But if it semantically fits from the REST descriptions, fine for me.. (We use 400 in such situations - but I am fine with it too)

Reply:
ah one more little thing: Shouldn't we name both these new exceptions with suffix Exception better? like ResourceAlreadyExistsException ?
I guess that is what most frameworks do too

Reply:
It's a good question indeed. I borrowed the idea from some frontend devs who seemed to like different status codes for different errors. I usually leave this to the frontend team to decide. Let's keep it for now and later we can change error messaging according to our needs.